README.adoc

```diff
@@ -0,0 +1,108 @@
+= Liquibase Runner
+
+--
+[WARNING]
+Older versions of this plugin may not be safe to use. Please review the following warnings before using an older version: +
+* https://jenkins.io/security/advisory/2018-03-26/#SECURITY-519[Plugin allows users to load arbitrary Java code into master JVM] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-2030[Missing permission check allows enumerating credentials IDs] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-1887[XXE vulnerability] +
+* https://www.jenkins.io/security/advisory/2020-09-23/#SECURITY-1885[Stored XSS vulnerability]
+--
+
+[.conf-macro .output-inline]#Provides https://www.liquibase.org/[Liquibase] build steps that evaluate liquibase changesets.#
```

Comment:
Would be nice to cleanup leftover macros

Reply:
Done

Reply:
Hi @oleg-nenashev ,
Not sure that need to update URL because it has already pointed on new Jenkins site.